application/src/objects/car-sprite.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var Phaser       = require('phaser');
+var rotateVector = require('../util').rotateVector;
+
+var Car = function(state, x, y, key)
+{
+    Phaser.Sprite.apply(this, [state.game, x, y, key]);
+
+    this.state = state;
+
+    this.state.game.physics.p2.enable(this);
+
+    this.body.mass = 10;
+};
+
+Car.prototype = Object.create(Phaser.Sprite.prototype);
+
+Car.prototype.accelerate = function()
+{
+    this.body.applyForce(
+        rotateVector(this.body.rotation, [0, 1600]),
+        this.body.x,
+        this.body.y
+    );
+};
+
+Car.prototype.brake = function()
+{
+    this.body.applyForce(
+        rotateVector(this.body.rotation, [0, -500]),
+        this.body.x,
+        this.body.y
+    );
+};
+
+Car.prototype.turnRight = function()
+{
+    this.body.rotateRight(80);
+};
+
+
+Car.prototype.turnLeft = function()
+{
+    this.body.rotateLeft(80);
+};
+
+Car.prototype.ROLLING_FRICTION_FORCE = 0.175;
+Car.prototype.SKID_FRICTION_FORCE    = 0.25;
+
+Car.prototype.applyForces = function()
```

Comment:
I discovered this yesterday: you can create an `update` method, and the state object will know to call that function in  the update phase of the game loop. So if you rename this function, you can get rid of that call in the state's update method.